imagelab/DisplayImage.java

```diff
@@ -94,5 +100,17 @@ public DisplayImage(ImgProvider imp, String title, boolean slow) {
         setJMenuBar(myMenuBar);
         setVisible(true);
     }
+    
+    public void remove(){
+        getContentPane().remove(pane);
+    }   
+    
+     public void synchronize(int w, int h, int[] pixs){
```

Comment:
This method directly addresses issue #2. The method is supplied portions of pixels from the pix array that is filled by pixel grabber in ImgProvider.java. The pixel data is first passed to the playTune method from Music.java that is called inside the play method in ImgProvider.java when the play menu item is selected within the user interface. 
This synchronize method is called for each iteration of the tuneIt iterator inside the playTune method in Music.java. The amount of rows of pixels that are passed to this method are incremented starting at one and increasing by the same amount for each chord so that the image finishes rendering at the same time the music stops playing. 
I will leave more comments about this process in the playTune method in the Music.java class.